packages/core/src/components/widgets/AltDateWidget.tsx

```diff
@@ -5,11 +5,13 @@ import {
   toDateString,
   pad,
   DateObject,
+  type DateElementFormat,
```

Comment:
I'm not sure the `type` is needed?
```suggestion
  DateElementFormat,
```